hal/lib/athena/Notifier.cpp

```diff
@@ -65,7 +65,7 @@ void updateNotifierAlarmInternal(std::shared_ptr<Notifier> notifier_pointer,
   if (triggerTime < closestTrigger) {
     closestTrigger = triggerTime;
     // Simply truncate the hardware trigger time to 32-bit.
-    notifierAlarm->writeTriggerTime((uint32_t)triggerTime, status);
+    notifierAlarm->writeTriggerTime((int32_t)triggerTime, status);
```

Comment:
tAlarm actually takes an `unsigned int`, so that's what we should be casting to.